components/match2/wikis/warcraft/match_group_input_custom.lua

```diff
@@ -46,12 +51,22 @@ local MapFunctions = {
 	ADD_SUB_GROUP = true,
 	BREAK_ON_EMPTY = true,
 }
+local FfaMatchFunctions = {
+	OPPONENT_CONFIG = {
+		resolveRedirect = true,
+		pagifyTeamNames = true,
+	},
+	readDate = MatchFunctions.readDate,
+}
+local FfaMapFunctions = {}
 
 ---@param match table
 ---@param options table?
 ---@return table
 function CustomMatchGroupInput.processMatch(match, options)
-	assert(not Logic.readBool(match.ffa), 'FFA is not yet supported in warcraft match2')
+	if Logic.readBool(match.ffa) then
+		return CustomMatchGroupInput.processFFAMatch(match, options)
+	end
```

Comment:
Is it possible to use the same logic as the other wikis do? Ie go though standardProccessMatch?